infrastructure/main.tf

```diff
@@ -10,103 +11,103 @@ locals {
 }
 
 data "azurerm_key_vault" "fees_key_vault" {
-  name = "${local.vaultName}"
+  name = local.vaultName
   resource_group_name = "${var.core_product}-${var.env}"
```

Comment:
Use join function to remove string interpolations

Reply:
Done the interpolation changes